### PR TITLE
Feature/trip dashboard

### PR DESCRIPTION
**Pull request by kiewi16**
Overview

- Adds login landing page. The authenticateLogin function that authenticates the user's username and password is currently not dynamic. It is presently hardcoded to be "traveler1" & "password". However, upon input of the correct username and password, the user is directed to the main page.
- Adds dynamic welcome message based on the user's name.
- Adds the destinations (city and country) the user has previously been to.
- Adds getUserData, getTripsTakenByUser, getDestinationsVisitedByUser, and calculateTotalSpentThisYear functions and related tests. There is presently no test for the calculateTotalSpentThisYear function. Additionally, the calculateTotalSpentThisYear function calculates the total the user has spent over their entire trip activity. Need to determine how I am going to define "current" year. Data ranges from 2019 to 2022. I will likely use the most "recent" year the user has data for and consider that the "current" year.

### CODE REVIEW

**Review by Sterling47 — APPROVED** [review]
HTML Structure Looks great. Nice use of Sematic Elements.

**Comment by kiewi16** [comment]
Adam reviewed PR request on Thursday, June 6th. We briefly discussed code via slack huddle.

File: dist/login-page.html
Comment (Sterling47) [inline dist/login-page.html:1]: Interesting that you made a second html file.

**Review by Sterling47 — APPROVED** [review]
This looks Great Kim! i love the use of mock data sets, semantic html and the use of iterator methods in your userFunctions.js file.

**Pull request merged**